docs/en/compute/Hive.md

```diff
@@ -337,7 +339,9 @@ You can run the following command in the Alluxio project directory:
 integration/checker/bin/alluxio-checker.sh hive -hiveurl [HIVE_URL]
 ```
 
-You can use `-h` to display helpful information about the command.
+You can use `-h` to display helpful information about the command, along with this detailed page on 
+[JDBC connection URLs](https://cwiki.apache.org/confluence/display/Hive/HiveServer2+Clients#HiveServer2Clients-JDBC) 
+within Hive.
```

Comment:
nit: remove "within Hive" and add "to specify the Hive URL"

Reply:
i'll update it now!